### PR TITLE
Sweat

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/diet.ts [hunk 1]
```diff
@@ -853,6 +868,14 @@ export function consumeDiet(diet: Diet<Note>, name: DietName): void {
             consumeSafe(countToConsume, menuItem.item, menuItem.additionalValue);
           },
         ],
+        [
+          $item`designer sweatpants`,
+          (countToConsume: number) => {
+            for (let n = 1; n <= countToConsume; n++) {
+              useSkill($skill`Sweat Out Some Booze`);
```
Comment (pstalcup) [inline src/diet.ts:875]: will this automatically equip the pants?

**Review by pstalcup — CHANGES_REQUESTED** [review]

File: src/diet.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/diet.ts:875]: yes

**Comment by Malibu-Stacey** [comment]
There's a bug with using the Sweat out some Booze skill via scripts (ASH at least, I expect JS also has the same issue).
It doesn't decrement the sweat property at all.
I've reported it on the mafia forum -> https://kolmafia.us/threads/casting-sweat-out-some-booze-via-use_skill-isnt-decrementing-the-sweat-property.27699/

File: src/diet.ts (hunk 1 above)
Comment (gausie) [reply src/diet.ts:879]: 😎

File: src/diet.ts [hunk 2]
```diff
@@ -302,7 +307,8 @@ function menu(): MenuItem<Note>[] {
    *  > js Item.all().filter((item) => item.fullness > 0 && item.name.indexOf("lasagna") > 0 && getIngredients(item)["savory dry noodles"]).join(", ")
    */
   const lasagnas = $items`fishy fish lasagna, gnat lasagna, long pork lasagna`;
-  const smallEpics = $items`meteoreo, ice rice`.concat([$item`Tea, Earl Grey, Hot`]);
+  // eslint-disable-next-line libram/verify-constants
```
Comment (gausie) [inline src/diet.ts:310]: What's all this then

File: src/post.ts [hunk 3]
```diff
@@ -70,16 +73,30 @@ function coldMedicineCabinet(): void {
...
   if (
     getRemainingStomach() > 0 &&
-    !globalOptions.noDiet &&
     (!globalOptions.yachtzeeChain || get("_garboYachtzeeChainCompleted", false))
   ) {
     consumeDiet(computeDiet().pantsgiving(), "PANTSGIVING");
   }
 }
 
+function fillSweatyLiver(): void {
```
Comment (gausie) [inline src/post.ts:85]: Gross!

File: src/diet.ts (hunk 2 above)
Comment (horrible-little-slime) [reply src/diet.ts:310]: I forgot we hadn't pushed that libram change yet

**Pull request merged**